apps/web/src/components/templates/in-app-editor/InAppEditorBlock.tsx

```diff
@@ -48,7 +48,7 @@ export function InAppEditorBlock({
   }
 
   return (
-    <InAppWidgetPreview>
+    <InAppWidgetPreview readonly>
```

Comment:
Should this be a fixed `true` value? or should it receive this from somewhere? If I get it right the preview will be always disabled instead of only when in readonly env.

Reply:
Yes. You are right. Fixed it.